build/build.js

```diff
@@ -83,15 +91,28 @@ function displayStats() {
 }
 
 
-
-async function fetchAndDisplayCharacter() {
+async function fetchAndDisplayCharacter(character) {
     // fetch the caracter from supabase
-
+    getCharacter();
     // if the character has a head, display the head in the dom
+    if (character.head)
+        headEl.style.backgroundImage = `url(../assets/${character.head}-head.png)`;
     // if the character has a middle, display the middle in the dom
+    if (character.middle)
+        middleEl.style.backgroundImage = `url(../assets/${character.middle}-middle.png)`;
     // if the character has a pants, display the pants in the dom
-    
+    if (character.pants)
+        bottomEl.style.backgroundImage = `url(../assets/${character.pants}-pants.png)`;
```

Comment:
the properties of the character object are `head`, `middle`, `bottom`, and `catchphrases`.  One 105 you would want to access with dot notation the `character.bottom` instead of `character.pants`